fork-runner/src/main/java/com/shazam/fork/PermissionUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.shazam.fork;
```

Comment:
Could we move this to the system package? That's where most of the adb dependent code is.

Reply:
👍